core/src/nusi/fixed-select.tsx

```diff
@@ -17,9 +17,10 @@ import { get } from 'lodash';
 
 const { Option, OptGroup } = Select;
 const FixedSelect = React.forwardRef((props, ref) => {
+  const {options: propsOptions, ...rest} = props;
   const options = props.children || get(props, 'options');
   return (
-    <Select ref={ref} getPopupContainer={(triggerNode) => triggerNode.parentElement as HTMLElement} {...props}>
+    <Select ref={ref} getPopupContainer={(triggerNode) => triggerNode.parentElement as HTMLElement} {...rest}>
```

Comment:
u should consider two scenarios
1. user pass options like `[{ label: 'A', value: 'A' }]`,
2. user pass options as React comp array `[<Option .../>]`